src/Share/Postgres/Causal/Queries.hs

```diff
@@ -14,8 +14,6 @@ module Share.Postgres.Causal.Queries
     expectPgNamespace,
     savePgNamespace,
     saveCausal,
-    loadNamespaceIdForCausal,
-    expectNamespaceIdForCausal,
```

Comment:
These don't fit the new traversal loader pattern, and already had a replacement, so I replaced all usages of these.